britnay268/INDIVIDUAL-ASSIGNMENT-Team-Roster-bg
# Private team/bg

## Description
A team can be marked public or private.
If a team is public, any authenticated user can view the team and its members along with creator details.

## Related Issue
#34 

## Motivation and Context
This change was implemented to allow users to allow other users to see their team if the make the group private.
The user that is viewing a public team but did not create it can only view the members in the team.
A user cannot delete or edit a team and members of the team that they did not create that is public.

## How Can This Be Tested?
A user can click teams in the Nav Bar to see the teams and they will be able to see the teams that are public and private.
The user can click view on a public team to view members of that team.


## Types of changes
